clients/java/src/main/java/io/camunda/zeebe/client/api/command/DeployResourceCommandStep1.java

```diff
@@ -107,9 +107,8 @@ interface DeployResourceCommandStep2
      *
      * <h1>Deploy resource command specifics</h1>
      *
-     * <p>When no {@code tenantId} is specified explicitly, the deployed resources are owned by all
-     * tenants using the {@link #SHARED_TENANT_IDENTIFIER} as long as multi-tenancy is enabled in
-     * the Zeebe cluster.
+     * <p>When no {@code tenantId} is specified explicitly, the deployed resources are rejected as
+     * long as multi-tenancy is enabled in the Zeebe cluster.
```

Comment:
🔧 This is not specific to the deploy resource command. We can remove the entire override IMO.

Reply:
Good point, I'll remove it. If anything changes during implementation, we can adjust it later.